addon-test-support/wait-until.js

```diff
@@ -5,7 +5,7 @@ export function waitUntil(callback, { timeout = 1000 } = {}) {
   return new RSVP.Promise(function(resolve, reject) {
     let value = run(callback);
     if (value) {
-      resolve(value);
+      return resolve(value);
```

Comment:
I'd rather move the `return` down below the invocation of `resolve`.  I do not believe the return value of the `Promise` constructor's callback argument actually has any meaning.

Specifically, can you structure it like:

```js
if (value) {
  resolve(value);
  return;
}
```